types/cloudinary-core.d.ts

```diff
@@ -434,6 +434,18 @@ export class PictureTag extends HtmlTag {
     static "new"(name: string, publicId: string, options?: Transformation.Options): PictureTag;
 }
 
+/**
+ * Creates an HTML (DOM) Image tag using Cloudinary as the source.
+ * @constructor SourceTag
+ * @extends HtmlTag
+ * @param {string} [publicId]
+ * @param {Object} [options]
```

Comment:
Used the same format as the ImageTag class (copy-paste), do you want me to add a more descriptive line like lines 724-725?